Userland/Libraries/LibWeb/Streams/ReadableStream.h

```diff
@@ -56,7 +56,9 @@ class ReadableStream final : public Bindings::PlatformObject {
     bool is_closed() const;
     bool is_errored() const;
     bool is_locked() const;
-    void set_stream_state(State value) { m_state = value; }
+
+    State state() const { return m_state; }
+    void set_state(State value) { m_state = value; }
```

Comment:
Can I convince you to send a spec PR instead updating all these manual `[[state]]` checks to use the common definitions in https://streams.spec.whatwg.org/#other-specs-rs-introspect? :)

Reply:
I can certainly give it a shot! Though it would introduce a weird asymmetry between `ReadableStream` and `WritableStream`. I guess the asymmetry already exists, so it's not that big of a deal. (not sure why these operations don't just also exist for `WritableStream` as well)